package cms_8120;
option java_package = "com.hikvision.finance.fms.cms.hpp.center";
option java_outer_classname = "ServerProto";
import "type/cmd.proto";

//服务表
message Server
{
    //必须字段
    optional int32 id                       = 1;    //在数据库中的流水号
    optional int32 nType                    = 2;    //服务类型
    optional string strIP                   = 3;    //服务IP
    optional int32 nCtrlPort                = 4;    //控制端口
    optional int32 nDataPort                = 5;    //数据端口
    optional string strUser                 = 6;    //服务名称
    optional string strComment              = 7;    //备注
    optional int32 nMultId                  = 8;    //ID
    optional int32 nMultVer                 = 9;    //版本
    optional int32 nMultSysId               = 10;   //系统ID
    optional string strConfig               = 11;   //配置信息
    optional int32 nonline                  = 12;   //在线状态（1表示在线，0表示不在线）
}

message ReqServer
{
    optional int32 cmd                      = 1;    // CMD_MDT_SERVER_STATUS--更新服务在线状态
	optional int32 id                       = 2;    // 在数据库中的流水号
	optional int32 nType                    = 3;    // 服务类型
    optional string strIP                   = 4;    // 服务IP
	//optional int32 nonline                  = 5;    // 在线状态（1表示在线，0表示不在线）
}

message ServerList
{
    optional int32 cmd                		= 1;    // CMD_DT_SERVER_LIST
    repeated Server server                  = 2;    // 服务列表
}
